tools/deploy.sh

```diff
@@ -6,28 +6,26 @@ DEPLOY_DIR="trash"
 SERVICE_NAME=""
 
 # Find deploy target and service name
-case $1 in
+case $GITHUB_REF in
 
-beta)
+"refs/heads/beta")
   DEPLOY_DIR="dev.api.fuelrats.com"
   SERVICE_NAME="fr-api_dev"
   ;;
 
-
-master)
+"refs/heads/main")
   DEPLOY_DIR="api.fuelrats.com"
   SERVICE_NAME="fr-api"
   ;;
 
-
 *)
   echo "Current branch is not configured for auto-deploy. skipping deployment..."
   exit 1
   ;;
 esac
 
 # Move built project files to server
-rsync -r --delete-after ./dist/ fuelrats@emmental.fuelrats.com:/var/www/$DEPLOY_DIR/
+rsync -rlz --delete-after ./ fuelrats@emmental.fuelrats.com:/var/www/$DEPLOY_DIR/
```

Comment:
for reference: 
* `-r` - recursive (directory) sync
* `-l` - maintain symlinks
* `-z` - compress
* `--delete-after` - delete old files, but only after transfer is complete.